core-java-modules/core-java-console/src/main/java/com/baeldung/tableformatter/TableFormatterExamples.java

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.tableformatter;
+
+import java.util.ArrayList;
+import java.util.Formatter;
+import java.util.List;
+
+import de.vandermeer.asciitable.AsciiTable;
+import dnl.utils.text.table.TextTable;
+
+public class TableFormatterExamples {
+
+    public static void main(String[] args) {
+        List<Student> students = getStudentRecords();
+        formatUsingJavaFormatter(students);
+        formatUsingPrintStream(students);
+        formatUsingJTextUtils();
+        formatUsingASCIITable(students);
+    }
+
+    private static List<Student> getStudentRecords() {
+        Student student1 = new Student(1, "Reham Muzzamil", "reham@gmail.com");
+        Student student2 = new Student(2, "John Mark", "john@mark.com");
+        Student student3 = new Student(3, "Abraham Dales", "ab@dales.com");
+        List<Student> students = new ArrayList<>();
+        students.add(student1);
+        students.add(student2);
+        students.add(student3);
+        return students;
+    }
+
+    private static void formatUsingJavaFormatter(List<Student> students) {
+        Formatter formatter = new Formatter();
+        formatter.format("%16s %16s %16s\n", "Id", "Full Name", "Email Address");
```

Comment:
When using Formatter you should be using %n instead of \n.

Reply:
Done @JavierPeris

Reply:
Done @JavierPeris